README.md

```diff
@@ -77,13 +77,16 @@ services:
 
 ### Node.js
 
-**Prerequisites**: Node.js, ffmpeg
+**Prerequisites**:
+* Node.js 16.x (16.x is recommended because of LTS. 17.x seems to make problems)
+* ffmpeg
 
 1. `git clone https://github.com/codetheweb/muse.git && cd muse`
 2. Copy `.env.example` to `.env` and populate with values
 3. I recommend checking out a tagged release with `git checkout v[latest release]`
 4. `yarn install` (or `npm i`)
 5. `yarn start` (or `npm run start`)
+	* `(npm run start&)` to run the bot 24/7 on a server
```

Comment:
```suggestion

```

There's a lot of ways to run this on a server besides just as a background process, like [PM2](https://pm2.keymetrics.io/). I'd rather not recommend anything specific here.